src/main/java/com/exactpro/th2/conn/ServiceListener.java

```diff
@@ -171,14 +165,14 @@ public void onEvent(IServiceProxy service, ServiceEvent serviceEvent) {
     }
 
     @NonNull
-    private ConnectivityMessage createConnectivityMessage(List<IMessage> messages, MessageID messageId) {
+    private ConnectivityMessage createConnectivityMessage(List<IMessage> messages, MessageID.Builder messageIdBuilder) {
         LOGGER.debug(
                 "On message: book name `{}`; direction '{}'; sequence '{}'; messages '{}'",
-                messageId.getBookName(),
-                messageId.getDirection(),
-                messageId.getSequence(),
+                messageIdBuilder.getBookName(),
+                messageIdBuilder.getDirection(),
+                messageIdBuilder.getSequence(),
```

Comment:
Maybe print via the TextFormat.shortDebugString method to print full msg-id in one line.

Reply:
done